pages/actors.html

```diff
@@ -11,6 +11,11 @@
       href="https://fonts.googleapis.com/css2?family=Karla:wght@300;400;600&family=Roboto+Slab:wght@200;400&display=swap"
       rel="stylesheet"
     />
+    <link 
+    rel="stylesheet" 
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
```

Comment:
Ładując Font Awesome w ten sposób, ładujemy cały kod i wszystkie ikonki, a tak naprawdę wykorzystujemy na stronie tylko dwie z nich. To dosyć mocno obciąża stronę (obniża jej performance). Zdecydowanie lepszym rozwiązaniem będzie jedno z poniższych:
1. Zrezygnowanie całkiem z Font Awesome i wrzucenie ikonek Facebooka i Twittera jako inline SVG
2. Skorzystanie z paczki z NPMa i załadowanie w JSie tylko tych dwóch ikonek, korzystając z tej metody: https://fontawesome.com/v5.15/how-to-use/javascript-api/other/tree-shaking